ui/preference/list_preference.go

```diff
@@ -122,7 +146,8 @@ func (lp *ListPreference) modal(gtx layout.Context) layout.Dimensions {
 
 	lp.optionsRadioGroup.Value = lp.currentValue
 
-	return lp.theme.Modal().Layout(gtx, w, 1050)
+	lp.modal = lp.theme.Modal()
```

Comment:
Move this to line 60 after this. https://github.com/planetdecred/godcr/blob/3ae6014390d309bc1bbf12da47e19947cfed63ae/ui/preference/list_preference.go#L59